libs/wingsdk/src/core/app.ts

```diff
@@ -64,6 +64,20 @@ export abstract class App extends Construct {
     return App.of(scope.node.scope);
   }
 
+  /**
+   * Loads the `App` class for the given target.
+   * @param target one of the supported targets
+   * @returns an `App` class constructor
+   */
+  public static for(target: string): any {
+    try {
+      // eslint-disable-next-line @typescript-eslint/no-require-imports
+      return require(`../target-${target}/app`).App;
```

Comment:
To be clear, the purpose of this PR is just to reduce the time of `require('@winglang/sdk')` right? This doesn't effect any JSII loading later in wingc (`.jsii` still considered all the peer deps to be submodules)?

Reply:
yes, not yet :-)